3_summarize/src/calculate_max_benthic_area.R

```diff
@@ -0,0 +1,17 @@
+
+calculate_max_benthic_area <- function(target_name, all_hypsos) {
+  
+  purrr::map(all_hypsos, function(hypso) {
+    sum(benthic_areas(hypso$H, hypso$A))
```

Comment:
I'm not sure what `enframe` does and wondering if it isn't simpler and easier to read to do
```r
purrr::map(names(all_hypsos), function(site_id){
    tibble(site_id = site_id, benthic_area = sum(benthic_areas(all_hypsos[[site_id]]$H, morphometry[[site_id]]$A)))
}) %>% purrr::reduce(bind_rows) %>% 
    readr::write_csv(target_name)
```

Reply:
`enframe` is a newish tibble function that takes a named vector and turns it into a two-column data.frame

Reply:
I disagree that what you have suggested is simpler, but it is more familiar.

Reply:
Can you elaborate?

Reply:
I kind of like this method more because I think it is slightly easier to read to keep the calculation isolated from the reshaping/formatting steps.